packages/features/tips/UpgradeTip.tsx

```diff
@@ -37,12 +33,14 @@ export function UpgradeTip({
 
   if (isParentLoading || isLoading) return <>{isParentLoading}</>;
 
+  const isDark = document.querySelector("html").classList.contains("dark");
```

Comment:
this is failing lint, possibly null

Reply:
`const isDark = Boolean(document.querySelector("html").classList.contains("dark"));`